db/schema.rb

```diff
@@ -30,9 +30,16 @@
     t.integer "source_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "access"
     t.index "url, md5((title)::text), md5(summary)", name: "index_news_on_url_and_title_and_summary", unique: true
   end
 
+  create_table "news_counters", force: :cascade do |t|
+    t.bigint "acessos"
```

Comment:
Eu tenho três problemas com isso daqui:

1) Ali em cima está escrito "access" e aqui está "acessos". Como estamos fazendo tudo em inglês já, eu acho que seria melhor mudar aqui. Porém...
2) Eu não entendi a utilidade dessa tabela. Ela tem alguma chave estrangeira? O que ela tem de diferente da outra? 
3) Reforço o que disse lá em cima: isso daqui não parece um sistema de notícias mais acessadas, e sim um sistema de up/downvote.